readme.md

```diff
@@ -68,6 +68,81 @@ The XMage server locates by default the configuration file from the current work
 to the desired location, for example `-Dxmage.config.path=config/otherconfig.xml`. The option can be set from the 
 XMageLauncher in `Settings > Java > Server java options`.
 
+### Server connection
```

Comment:
Server configuration details must be placed to [wiki page](https://github.com/magefree/mage/wiki) (readme already contains link to wiki and it should not be overloaded with unnecessary information).

![shot_201226_153111](https://user-images.githubusercontent.com/8344157/103150639-958eef00-478f-11eb-95bf-a609ac8bc003.png)